manon/js/collapsible.js

```diff
@@ -1,118 +1,85 @@
 // @ts-check
 
-import { ensureElementHasId, onMediaQueryMatch, onDomReady } from "./utils.js";
+import { ensureElementHasId, onDomReady } from "./utils.js";
 
 onDomReady(initCollapsible);
 
-/**
- * Add responsive behaviour to collapsible components. Safe to call again to
- * make a newly added collapsible components.
- */
 export function initCollapsible() {
```

Comment:
```suggestion
/**
 * Finds and initializes all collapsible components on the page.
 *
 * This function scans the DOM for elements with the `.collapsible` class and
 * makes them interactive by creating and injecting a toggle button.
 * It is safe to call this function multiple times, as it will not
 * re-initialize a component that already has a toggle button.
 */
export function initCollapsible() {
```